database/migrations/1547727772_create_scale_options.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `scale_options` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
I guess this shouldn't be AUTO_INCREMENT as the value for this column is always set to the ID of the cluster the scale option  belongs to.